src/main/resources/react4xp/dashboard/containers/Jobs/index.jsx

```diff
@@ -212,7 +214,10 @@ export function Jobs() {
           </React.Fragment>
         )
       })
-    } else if (currentModalJob.task === 'Refresh dataset calculators') {
+    } else if (
```

Comment:
To avoid duplicate code (caused by me) you can also include currentModalJob.task === 'Refresh dataset' from above since its same code